src/main/java/io/wispforest/owo/ui/core/OwoUIDrawContext.java

```diff
@@ -154,6 +156,14 @@ public void drawText(Text text, float x, float y, float scale, int color) {
     }
 
     public void drawText(Text text, float x, float y, float scale, int color, TextAnchor anchorPoint) {
```

Comment:
Not so sure whether these two overloads are necessary, we're talking about a single extra parameter

Reply:
These methods are overloaded to provide backwards compatibility. Introducing a required parameter in both `drawText(Text text, float x, float y, float scale, int color, TextAnchor anchorPoint)` and `drawText(Text text, float x, float y, float scale, int color)` would break the existing method calls for no good reason